packages/attribute-registry-process/src/model/domain/errors.ts

```diff
@@ -65,7 +65,7 @@ export function OrganizationIsNotACertifier(
   tenantId: TenantId
 ): ApiError<ErrorCodes> {
   return new ApiError({
-    detail: `Tenant ${tenantId} is not a Certifier`,
+    detail: `Organization ${tenantId} is not a Certifier`,
     code: "OrganizationIsNotACertifier",
     title: "Organization is not a certifier",
```

Comment:
Since these details are exposed to the requester, we should be consistent with tenant references
```suggestion
    detail: `Tenant ${tenantId} is not a Certifier`,
    code: "OrganizationIsNotACertifier",
    title: "Tenant is not a certifier",
```